src/main/java/com/google/starfish/models/User.java

```diff
@@ -0,0 +1,45 @@
+package com.google.starfish.models;
+
+import java.util.Date;
+
+/** Represents a user */
+public class User {
+
+  private long id;
+  private String displayPicture;
+  private String displayName;
+  private Date dateJoined;
+  private String email;
+  private long points;
+  private String school;
+  // Array of noteIds that have have been favorited by the user
+  private long[] favoriteNotes;
+
```

Comment:
I believe we might need an array for uploadedNotes or submittedNotes.

Reply:
I think you're right, I'll push that change!